lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -293,45 +330,71 @@ def commit(action, post, authorization = nil)
 
         succeeded = success_from(action, response)
 
-        response(action, succeeded, response)
+        response(action, succeeded, response, source_id)
       end
 
-      def response(action, succeeded, response)
+      def response(action, succeeded, response, source_id = nil)
         successful_response = succeeded && action == :purchase || action == :authorize
         avs_result = successful_response ? avs_result(response) : nil
         cvv_result = successful_response ? cvv_result(response) : nil
 
+        if action == :unstore
+          message_text = "The instrument #{source_id} has been deleted/unstored"
+          authorization = nil
```

Comment:
💬 Comment:

You don't need to set `authorization` to nil, Ruby is going to do that for you.

Reply:
Thanks